### PR TITLE
[GNA] [speech_sample] Fix accuracy problem when nthreads == 2

### PR DESCRIPTION
**Pull request by kbruniec**
### Details:
Fix accuracy problem when nthreads == 2
  - breaking commit:
     https://github.com/openvinotoolkit/openvino/commit/aff7a6608286129cb7eb9019854959c0c155718e

### Tickets:
 - CVS-49036

### CODE REVIEW

File: inference-engine/samples/speech_sample/main.cpp [hunk 1]
```diff
@@ -1029,6 +1029,11 @@ int main(int argc, char *argv[]) {
                             continue;
                         }
 
+                        ptrInputBlobs.clear();
+                        for (auto& input : cInputInfo) {
+                            ptrInputBlobs.push_back(inferRequest.inferRequest.GetBlob(input.first));
+                        }
+
```
Comment (admitriev-gna) [inline inference-engine/samples/speech_sample/main.cpp:1036]: @kbruniec , Will this correct work with option -iname?

Comment (kbruniec) [reply inference-engine/samples/speech_sample/main.cpp:1036]: How to test it?
As you are experienced with this, maybe you could check?

Comment (admitriev-gna) [reply inference-engine/samples/speech_sample/main.cpp:1036]: Ok, I will check )

Comment (dorloff) [reply inference-engine/samples/speech_sample/main.cpp:1036]: @admitriev-gna, do we have a test which checks nthreads == 2 together with -iname option?

Comment (admitriev-gna) [reply inference-engine/samples/speech_sample/main.cpp:1036]: No, we haven't such as test.

Comment (admitriev-gna) [reply inference-engine/samples/speech_sample/main.cpp:1036]: @kbruniec I created draft PR with supporting -iname and your fix
https://github.com/openvinotoolkit/openvino/pull/4774

Comment (kbruniec) [reply inference-engine/samples/speech_sample/main.cpp:1036]: @admitriev-gna Thanks I am reviewing it.

@admitriev-gna @dorloff  is "-iname" documented somewhere? Do we have a test for iname?

I can not find it on https://docs.openvinotoolkit.org/latest/openvino_inference_engine_samples_speech_sample_README.html

**Pull request closed without merging**